modules/core/04-channel/keeper/upgrade.go

```diff
@@ -81,15 +81,30 @@ func (k Keeper) ChanUpgradeTry(
 	return types.Upgrade{}, nil
 }
 
-// WriteUpgradeTryChannel writes a channel which has successfully passed the UpgradeTry step.
+// WriteUpgradeTryChannel writes a channel which has successfully passed the UpgradeTry handshake step.
 // An event is emitted for the handshake step.
 func (k Keeper) WriteUpgradeTryChannel(
 	ctx sdk.Context,
 	portID, channelID string,
 	proposedUpgrade types.Upgrade,
+	flushStatus types.FlushStatus,
```

Comment:
What about writing the channel flush status as FLUSHING, and then in a followup pr we add a function call to check if flushing is completed?

Reply:
it's an interesting idea, i guess it depends on when we want to call the  `pendingInFlightPackets` function. i think we should discuss when we get to the `msg_server` implementation, we can modify it then if need be.

Reply:
Let's discuss tomorrow